File: 2017-10-04 Més Python/p2.py
from jutge import read

# La funció maxim, donats dos enters retorna un enter que és el seu màxim.
def maxim(a, b):
    if a > b:
        return a
    else:
        return b
    
# La funció maxim3, donats tres enters retorna un enter que és el seu màxim.
def maxim3(a, b,c):
    return maxim(maxim(a, b), c)

# Programa que llegeix tres enters i n'escriu el seu màxim, cridant maxim3().    
x, y, z = read(int, int, int)
print(maxim3(x, y, z))

File: 2017-09-22 Funcionem!/p2.cc
#include <iostream>
using namespace std;


// La funció maxim, donats dos enters retorna un enter que és el seu màxim.
int maxim(int a, int b) {
    if (a > b) return a;
    else return b;
}


// La funció maxim3, donats tres enters retorna un enter que és el seu màxim.
int maxim3(int a, int b, int c) {
    return maxim(a, maxim(b, c));
}


// Programa que llegeix tres enters i n'escriu el seu màxim, cridant maxim3().
int main() {
    int x, y, z;
    cin >> x >> y >> z;
    cout << maxim3(x, y, z) << endl;
}